FreeRTOS-Plus/Demo/coreSNTP_Windows_Simulator/readme.url

```diff
@@ -0,0 +1,5 @@
+[{000214A0-0000-0000-C000-000000000046}]
+Prop3=19,11
+[InternetShortcut]
+IDList=
+URL=https://www.freertos.org/sntp/sntp-demo.html
```

Comment:
This link does not work.

Reply:
It is primarily a placeholder for now. It will be updated when documentation is updated.